lightkurve/correctors/pldcorrector.py

```diff
@@ -85,52 +84,151 @@ class PLDCorrector(Corrector):
         (arXiv:1702.05488)
     .. [4] EVEREST pipeline webpage, https://rodluger.github.io/everest
     """
-    def __init__(self, tpf):
+
+    def __init__(self, tpf, aperture_mask=None):
+
         self.tpf = tpf
+        if aperture_mask is None:
+            aperture_mask = tpf.create_threshold_mask(2)
+        self.lc = self.tpf.to_lightcurve(aperture_mask=aperture_mask)
 
-        self.flux = tpf.flux
-        self.flux_err = tpf.flux_err
-        self.time = tpf.time.value
+        super(PLDCorrector, self).__init__(lc=self.lc)
```

Comment:
Now that we only support Python 3, we can simply call `super()`, like this:
```suggestion
        super().__init__(lc=self.lc)
```